src/compose/ComposeBox.js

```diff
@@ -75,13 +76,19 @@ type SelectorProps = {|
   stream: Subscription | {| ...Stream, in_home_view: boolean |},
 |};
 
-type Props = $ReadOnly<{|
-  insets: EdgeInsets,
-
+type OuterProps = $ReadOnly<{|
   narrow: Narrow,
   editMessage: EditMessage | null,
   completeEditMessage: () => void,
+|}>;
 
+type Props = $ReadOnly<{|
+  ...OuterProps,
+
+  // from withSafeAreaInsets
+  insets: EdgeInsets,
+
+  // from `connect`
```

Comment:
nit: Let's put these (and the prop from `withGetText`, below) in a consistent order relative to how we do the wrapping, so it's easy to compare the two and see that they line up.

In the common case with just `connect`, we generally put those at the end, so I guess let's do it consistent with that: `OuterProps` first, then the next thing inward (which is `withGetText`), then the next thing inward, etc.